.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,17 +4,19 @@
 
 ### How do I test this PR?
 
+---
+
 > I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
 
-## TODO:
+## TODO
 
 - [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
-- [ ] Closes [#ISSUE]
+- [ ] Closes [DEV-XXX](url)
```

Comment:
This creates the `DEV-XXX` as a link that doesn't go anywhere. Maybe change the formatting so that it doesn't create the URL but it does suggest that what you're looking for here is a URL to the Jira issue. 

Also, I just wanted to note that when you put in a Github issue number, this actually closes the Github issue. This probably doesn't _actually_ move the Jira issue to `Done` or anything. Man ... if only it could. I get the idea though ... just putting in a reference to the task this PR was for is smart. 👍